Solution1/Solution1.Tests/WeatherApi/WeatherControllerTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using BusinessLayer.Command;
+using BusinessLayer.Command.Abstract;
+using BusinessLayer.Configuration.Abstract;
+using BusinessLayer.DTOs;
+using BusinessLayer.Services.Abstract;
+using KellermanSoftware.CompareNetObjects;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using WeatherApi.Controllers;
+using Xunit;
+
+namespace Weather.Tests.WeatherApi
```

Comment:
tests should be located in folders, not in one heap, it's not uncomfortable, is it?
folders should duplicate same structure like in project you test

Reply:
+